### ISSUE TITLE
Suggestion - Iris System Update

### ISSUE THREAD

**Issue by JayCoolGuy** [StargateMC/IssueTracker#400]
## What is your suggestion?
> I believe that the iris system should be updated.  Specifically in the way that normal irises have a durability, and after a certain amount of damage, they break.  Perhaps there could also be a type of device used for taking down irises specifically, such as a laser of some type, perhaps the mekanism laser.  I also think that there should be some type of higher tier of iris, an energy iris like on the Atlantis gate.  I think this iris should be different in the way of not having a higher durability but, in only lasting according to the amount of power is being inputted into it.  So if it were to run out of energy, then the energy iris would shut down.

## How will this benefit players?
>Players will now not be immediately stopped by the indestructible, easy-to-obtain, normal iris and will now still have to work to bring down an iris but will know that it is at least possible.  I think that this update will make the game more interesting, now that being safe is harder than previously.  It'll add more fun to the game and make people think more strategically. 

## How will this change for players?
>Players who aren't that advanced will have to be weary knowing that their iris can break, and should probably make some other type of defense.  More advanced players will now be the ones in control of their safety, being they control how much power they make which in turn would decide how long their energy iris will stay active.  This also makes raiding people with an iris a possibility which will make things better in my opinion. 

## Do you consider this suggestion a:
- [x] Should (Beneficial to players, but can do without)

**Comment by luckyrada** [comment]
Yes this sounds really cool I think it would be awesome. Especially the laser xD

**Comment by Joey4213** [comment]
#305 #209
